cmd/mqtt/main.go

```diff
@@ -85,6 +86,13 @@ const (
 	defESURL  = "localhost:6379"
 	defESPass = ""
 	defESDB   = "0"
+	// AUTH
+	envAuthURL  = "MF_MQTT_ADAPTER_AUTH_URL"
+	envAuthPass = "MF_MQTT_ADAPTER_AUTH_PASS"
+	envAuthDB   = "MF_MQTT_ADAPTER_AUTH_DB"
+	defAuthURL  = "localhost:6379"
```

Comment:
This is actually Cache, and should be called like this IMHO - to avoid confusion with Auth service